designs/native-ami-versioning.md

```diff
@@ -0,0 +1,121 @@
+# Native AMI Versioning
+
+Karpenter uses the latest EKS Optimized AMI releases as defaults in their AMI Requirements. With [Drift](https://karpenter.sh/preview/concepts/deprovisioning/#drift) implemented, when a new EKS Optimized AMI is released users who have enabled drift could see their fleet rolled out to the newly released AMI without warning.
```

Comment:
The sentence structure is a bit confusing here.  I think it's worth calling out the different "EKS Optimized" AMIs in this intro section too since that can be another point of confusion on exactly what we're talking about here. 

```suggestion
Karpenter uses the latest EKS Optimized AMI releases for Bottlerocket, Amazon Linux 2, and Ubuntu as the defaults for AMI Requirements. With [Drift](https://karpenter.sh/preview/concepts/deprovisioning/#drift) implemented, when a new EKS Optimized AMI is released, users who have enabled drift could see the newly released AMIs rolled out to their fleet without warning.
```